example.json

```diff
@@ -2,8 +2,10 @@
   "DisclosureId": "66071ef0-9dbe-46d0-a0b0-e5d238fc2683",
   "Date": "2018-07-07",
   "RuleCode": "MajorAR",
-  "Issuer": "Banco Argentino",
-  "Entity": "Hedge Fund Inc",
+  "IssuerName": "Banco Argentino",
+  "IssuerLEI": "579100KKDDKIFCBKB036",
+  "Entity": "UBS Life UK (World) Equity Tracker Fund",
```

Comment:
@ApwFA very minor comment. This looks inconsistent 🔵: `IssuerName` for Issuer but not for `Entity`?